test/deploy-pipeline-mkp-cli.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+echo "=================START MKP DEPLOY================"
+
+GCR_IMAGE_TAG=${GCR_IMAGE_TAG:-latest}
+
+# Configure gcloud as a Docker credential helper
+gcloud auth configure-docker
+
+# Delete argo first because KFP comes with argo too
+kubectl delete namespace argo --wait || echo "No argo installed"
+
+# Install the application resource definition
+kubectl apply -f "https://raw.githubusercontent.com/GoogleCloudPlatform/marketplace-k8s-app-tools/master/crd/app-crd.yaml"
+
+# Grant user ability for using Role-Based Access Control
+kubectl create clusterrolebinding cluster-admin-binding \
+  --clusterrole cluster-admin \
+  --user $(gcloud config get-value account)
+
+export APP_INSTANCE_NAME=kubeflow-pipelines-test
+
+# Install mpdev
+BIN_FILE="$HOME/bin/mpdev"
+if ! which mpdev; then
+  echo "Install mpdev"
+  docker pull gcr.io/cloud-marketplace-tools/k8s/dev
+  mkdir -p $HOME/bin/
+  touch $BIN_FILE
+  export PATH=$HOME/bin:$PATH
+  docker run gcr.io/cloud-marketplace-staging/marketplace-k8s-app-tools/k8s/dev:remove-ui-ownerrefs cat /scripts/dev > "$BIN_FILE"
```

Comment:
From [this recent change](https://github.com/kubeflow/pipelines/commit/fe6cf9a1f2dd19070c125741c63539f945e89b53) it seems like MKP already support owner ref removing in their official release, shall we consider also using their official release here (if they have)?

/cc @rmgogogo  can you take a look on this？ Thanks

Reply:
Let us put this into a remaining work item in #2864